api/StaticRange.json

```diff
@@ -54,16 +54,16 @@
           "description": "<code>StaticRange()</code> constructor",
           "support": {
             "chrome": {
-              "version_added": "60"
+              "version_added": null
```

Comment:
Chrome, Chrome Android, and WebView Android should say `false`.

Reply:
> Chrome, Chrome Android, and WebView Android should say `false`.

OK, thanks — made that change